C/plugins/storage/sqlite/connection.cpp

```diff
@@ -2202,8 +2202,8 @@ int blocks = 0;
 
 	result = "{ \"removed\" : 0, ";
 	result += " \"unsentPurged\" : 0, ";
-	result += " \"unsentRetained\", 0, ";
-    	result += " \"readings\" 0 }";
+	result += " \"unsentRetained\" : 0, ";
+    	result += " \"readings\" : 0 }";
```

Comment:
I'm surprised, we could never catch this. I'm sure json parser should have failed with this.

Reply:
https://github.com/foglamp/FogLAMP/pull/1366/commits/e666cf799164bc6cbeeddc5f1be8e1f935f33bf5

Looks like we were assigning new (string) values on actual purge. and when we had no rows to purge, there was bare exception raised by plugin,  hence parser purge process parser never read these default.